src/test/regress/citus_tests/arbitrary_configs/citus_arbitrary_configs.py

```diff
@@ -76,20 +75,25 @@ def run_for_config(config, lock, sql_schedule_name):
         common.save_regression_diff("postgres", config.output_dir)
 
     exitCode |= _run_pg_regress_on_port(
```

Comment:
very minor: can you maybe change the indentation to match the below `sql_schedule_name`? That'd have made my reading easier